pkg/auth/providers/saml/saml_client.go

```diff
@@ -394,7 +397,7 @@ func (s *Provider) HandleSamlAssertion(w http.ResponseWriter, r *http.Request, a
 	if err != nil {
 		log.Error(err)
 		// UI uses this translation key to get the error message
-		http.Redirect(w, r, redirectURL+"errorCode=422&errorMsg="+UITranslationKeyForErrorMessage, http.StatusFound)
+		http.Redirect(w, r, redirectURL+"errorCode=422&err="+UITranslationKeyForErrorMessage, http.StatusFound)
```

Comment:
What does this change affect? Is it causing the wrong re-direct too?

Reply:
It looks like the UI team changed the `errorMsg` key to `err`. So now, if we send the error message with `errorMsg`, it doesn’t show up on the UI, but if we send it as `err`, it works and displays fine.

Reply:
Awesome! Good catch!